README.md

```diff
@@ -27,8 +29,244 @@ runApp(...
 
 ```
 
-2. Run the application in debug mode and watch for a leak related warnings. If you see a warning, open the link to investigate the leaks.
+2. Run the application in debug mode and watch for a leak related warnings.
+If you see a warning, open the link to investigate the leaks.
+
+TODO(polina-c): implement the link and add example of the warning.
+
+### Flutter tests
+
+Wrap your tests with `withLeakTracking` to setup automated leak verification:
+
+```dart
+test('...', () async {
+  final leaks = await withLeakTracking(
+    () async {
+      ...
+    },
+  );
+
+  expect(leaks, leakFree);
```

Comment:
What is `leaksFree`? I can't seem to find a reference to it in the repo

Reply:
Will show up in the next PR